pegasus/sites.v3/code.org/public/educate/gamelab.haml

```diff
@@ -0,0 +1,73 @@
+
+  - csdunit3_videos = []
+  - csdunit3_videos << { id: 'csd_gamelab_draw_1', category_title: 'Drawing in Game Lab', code: 'PXn9gKiKKFo', download_path: 'https://videos.code.org/cs-discoveries/gamelab/gamelab_drawing_1.mp4', studio_stage: '/s/csd3/stage/3/puzzle/2' }
+  - csdunit3_videos << { id: 'csd_gamelab_draw_2', code: 'QLabeWhw_O0', download_path: 'https://videos.code.org/cs-discoveries/gamelab/gamelab_drawing_2.mp4', studio_stage: '/s/csd3/stage/3/puzzle/4' }
+  - csdunit3_videos << { id: 'csp_applab_variables_1', category_title: 'Introduction to variables', code: 'G41G_PEWFjE', download_path: 'https://videos.code.org/2015/csp/applab/variables_1.mp4', studio_stage: '/s/csd3/stage/5/puzzle/2' }
+  - csdunit3_videos << { id: 'csd_gamelab_sprites_1', category_title: 'Animation sprites', code: 'aUHzpGt9nso', download_path: 'https://videos.code.org/cs-discoveries/gamelab/gamelab_sprites.mp4', studio_stage: '/s/csd3/stage/6/puzzle/2' }
+  - csdunit3_videos << { id: 'csd_gamelab_sprites_2', code: 'G666sFzAg5g', download_path: 'https://videos.code.org/cs-discoveries/gamelab/gamelab_animating_sprites.mp4', studio_stage: '/s/csd3/stage/9/puzzle/2' }
+  - csdunit3_videos << { id: 'csd_gamelab_animation_tab', category_title: 'Animation tab', code: 'DQPruYTJ0Fk', download_path: 'https://videos.code.org/cs-discoveries/gamelab/gamelab_animation_tab.mp4', studio_stage: '/s/csd3/stage/6/puzzle/7' }
+  - csdunit3_videos << { id: 'csd_gamelab_drawloop', category_title: 'Draw loop', code: 'G6QJeuHhqCM', download_path: 'https://videos.code.org/cs-discoveries/gamelab/gamelab_drawloop.mp4', studio_stage: '/s/csd3/stage/7/puzzle/2' }
+  - csdunit3_videos << { id: 'csp_applab_conditionals_1', category_title: 'Conditionals', code: 'y3rCKJNOwpA', download_path: 'https://videos.code.org/2015/csp/applab/conditionals_1.mp4', studio_stage: '/s/csd3/stage/11/puzzle/3' }
+  - csdunit3_videos << { id: 'csp_applab_conditionals_2', code: 'UDi7xgIIW8E', download_path: 'https://videos.code.org/2015/csp/applab/conditionals_2.mp4', studio_stage: '/s/csd3/stage/13/puzzle/4' }
+  - csdunit3_videos << { id: 'csd_gamelab_velocity', category_title: 'Sprite velocity', code: 'zcP94yMYqrI', download_path: 'https://videos.code.org/cs-discoveries/gamelab/gamelab_velocity.mp4', studio_stage: '/s/csd3/stage/15/puzzle/2' }
+
+  - csdunit3_videos.each do |video|
+    // Show the chapter title at the
+    // beginning of each video chapter
+    - unless video[:category_title].nil_or_empty?
+      .clear
+      %h2= video[:category_title]
+    %div{style: "float:left; padding:10px; width: 275px;"}
+      = view :display_video_thumbnail, id: video[:id],  video_code:video[:code], caption: video[:caption], play_button: 'center', download_path: video[:download_path], letterbox: 'false'
+      - unless video[:studio_stage].nil_or_empty?
+        %a{href: CDO.studio_url(video[:studio_stage])} Try it now
+
+:javascript
+  $(function() {
+    $('.highlight-item').mouseover(function() {
```

Comment:
maybe a comment about what exactly is fading in & out?  also do we need a touch equivalent for non-mouse users?